pom.xml

```diff
@@ -63,23 +63,23 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.7.1</jenkins.version>
```

Comment:
Ok, I'll bump it to 2.7.3 - neither BO nor Declarative depend on `workflow-aggregator`, so they're not going to get messed up by bumping the aggregator core dependency.